<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileEdit</class>
 <widget class="QDialog" name="ProfileEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Turn off after</string>
       </property>
       <property name="buddy">
        <cstring>off</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QCheckBox" name="awake">
       <property name="text">
        <string>Awake screen upon activation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="suspendNever">
       <property name="text">
        <string>never</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="off">
       <property name="suffix">
        <string> min</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1089</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="offNever">
       <property name="text">
        <string>never</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="standbyNever">
       <property name="text">
        <string>never</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Suspend after</string>
       </property>
       <property name="buddy">
        <cstring>suspend</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Standby after</string>
       </property>
       <property name="buddy">
        <cstring>standby</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="suspend">
       <property name="suffix">
        <string> min</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1089</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="standby">
       <property name="suffix">
        <string> min</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1089</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>name</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>standby</tabstop>
  <tabstop>standbyNever</tabstop>
  <tabstop>suspend</tabstop>
  <tabstop>suspendNever</tabstop>
  <tabstop>off</tabstop>
  <tabstop>offNever</tabstop>
  <tabstop>awake</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProfileEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProfileEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>standbyNever</sender>
   <signal>toggled(bool)</signal>
   <receiver>standby</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>suspendNever</sender>
   <signal>toggled(bool)</signal>
   <receiver>suspend</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offNever</sender>
   <signal>toggled(bool)</signal>
   <receiver>off</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
